### PR TITLE
Use different color sheme

### PR DESCRIPTION
**Pull request by chanteriam**
## How was it tested?
- [ ] Wrote test cases.
- [x] Previously working test cases are still passing.

```
================== test session starts ==================
platform darwin -- Python 3.11.1, pytest-7.2.1, pluggy-1.0.0 -- /Users/shaymilner/Library/Caches/pypoetry/virtualenvs/reproductive-rights-data-project-4RXIkUiy-py3.11/bin/python
cachedir: .pytest_cache
rootdir: /Users/shaymilner/Documents/Harris/01Winter23/CAPP_II/final_project/reproductive-rights-data-project
plugins: dash-2.8.1
collected 13 items                                      

reproductive_rights_data_project/test/test_api.py::test_get_api_data PASSED
reproductive_rights_data_project/test/test_api.py::test_get_state_zip_code_geo_json_first_call PASSED
reproductive_rights_data_project/test/test_api.py::test_get_state_zip_code_geo_json_not_first_call PASSED
reproductive_rights_data_project/test/test_api.py::test_set_default_types PASSED
reproductive_rights_data_project/test/test_api.py::test_fill_in_missing_data PASSED
reproductive_rights_data_project/test/test_api.py::test_add_missing_states PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_make_row_dicts PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_set_default_types PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_clean_ansirh PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_translate_code_to_state PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_split_by_state PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_split_by_zip PASSED
reproductive_rights_data_project/test/test_util.py::test_to_json PASSED
```
- [x] I ran `pylint` and have made the bulk of changes that it has requested.
- [x] I ran the `black` formatter and pushed the changes.

### Here's what the color palette looks like:
Screen Shot 2023-03-05 at 22 24 15

Screen Shot 2023-03-05 at 22 24 59

### CODE REVIEW

File: visualization/app.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+# test function for visualization
+def read_state():
+    """
+    Test functon that reads in the states.txt data for the state dropdown
+        nav bar
+
+    Returns:
+        (lst) list of states in alphabetical order
+    """
+    with open("./data/states.txt") as state:
```
Comment (michplunkett) [inline visualization/app.py:27]: There is a constant for this.

File: visualization/app.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+from charts.state_summary import StateSummary as StateSummary
+
+DASH_INSTANCE = Dash(__name__, external_stylesheets=[dbc.themes.LUX])
+
+
+# test function for visualization
+def read_state():
+    """
+    Test functon that reads in the states.txt data for the state dropdown
+        nav bar
```
Comment (michplunkett) [inline visualization/app.py:22]: You don't need to tab this part and throw a period on that bb.

File: visualization/app.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
+"""
+Create Plotly Dash Visualization to map the cleaned data
+
+Author(s): Aïcha Camara
+"""
+import plotly.graph_objects as go
```
Comment (michplunkett) [inline visualization/app.py:6]: This directory isn't in the application.

**Comment by chanteriam** [comment]
Here's what it looks like:
image

image

File: visualization/app.py (hunk 3 above)
Comment (chanteriam) [reply visualization/app.py:6]: @necabotheking Because idk what that means

Comment (michplunkett) [reply visualization/app.py:6]: You added a whole other `app.py` file in a folder that didn't exist prior.

File: reproductive_rights_data_project/visualization/charts/zip_code.py [hunk 4]
```diff
@@ -114,13 +114,14 @@ def create_visual(self):
                 go.Table(
                     header=dict(
                         values=list(zip_df.columns),
-                        fill_color="silver",
+                        fill_color="#300608",
+                        font_color="#E0DFDF",
```
Comment (michplunkett) [inline reproductive_rights_data_project/visualization/charts/zip_code.py:118]: Update the `Author(s)`.

File: reproductive_rights_data_project/visualization/charts/state_summary.py [hunk 5]
```diff
@@ -192,13 +192,16 @@ def create_visual(self):
                 go.Table(
                     header=dict(
                         values=list(state_summary_df.columns),
-                        fill_color="silver",
+                        fill_color="#300608",
+                        font_color="#E0DFDF",
```
Comment (michplunkett) [inline reproductive_rights_data_project/visualization/charts/state_summary.py:196]: Update the `Author(s)`

**Review by necabotheking — APPROVED** [review]
Looks good!

**Pull request merged**